graspologic/models/rdpg.py

```diff
@@ -86,11 +89,11 @@ class RDPGEstimator(BaseGraphEstimator):
 
     def __init__(
         self,
-        loops=False,
-        n_components=None,
-        ase_kws={},
-        diag_aug_weight=1,
-        plus_c_weight=1,
+        loops: bool = False,
+        n_components: Optional[int] = None,
+        ase_kws: Dict[str, Any] = {},
+        diag_aug_weight: float = 1,
```

Comment:
does this not break with a 1 (int) default value? if so, im just curious as to why

Reply:
No! [PEP 484](https://www.python.org/dev/peps/pep-0484/) places float and int into a pseudo-inheritance hierarchy for the purposes of type hinting:

```
Rather than requiring that users write import numbers and then use numbers.Float etc., this PEP proposes a 
straightforward shortcut that is almost as effective: when an argument is annotated as having type float, an argument of 
type int is acceptable; similar, for an argument annotated as having type complex, arguments of type float or int are 
acceptable.
```

Reply:
that is an amazing feature IMO, thank you!